docs/proposals/secret-manager-store-api.md

```diff
@@ -0,0 +1,70 @@
+# Secrets Store Proposal
```

Comment:
The title and the filename don't seem to be relevant any longer, could you please align with the actual content of the proposal?

Reply:
Done